### ISSUE TITLE
[Midterm_Project_Credit-Card-Marketing_Classification] Kathi

### ISSUE THREAD

**Issue by katharina-beriault** [Ironhack-Data-0621-Remote/mid-bootcamp-project#12]
https://github.com/katharina-beriault/Midterm_Project_Credit-Card-Marketing_Classification

**Comment by Fominayasg** [comment]
# Hi Kathi 😁 !!

Let's have a look at your project. 

First of all I want to congratulate you because of the folder organization and the README file. They are gorgeus!!

The only thing is that you duplicated the csv file in "Python" folder and in "data" folder.

You also separated your functions into a `.py` file!!

## SQL

First of all thanks for the markdown file, you made my job way easier :heart:.

The queries are well executed and you explained some ideas about the data in the .md file which is realy useful. I have only a tip for you:

### Question 9 

you used SUM// count instead of AVG for requesting the average balance. Like this is easier:
```sql
select round(avg(Average_Balance),2) from credit_card_data;
```
and you can apply it to the next question too.

## EDA & Data cleaning
Really good job with the exploratory analysis!! You came up with some interesting functions that I'm sure you will reuse for other projects.

Maybe you can refine de `showing_count_dist` function so it shows only one plot per feature instead of one plot and an empty graphic. Also it would be nice if you comment each plot so the reader can understand what it means.

## Data preprocessing & ML

This part of the project is execellent, the only thing I can say is that when you apply your model and in some cases it says that the customer is always going to decline the offer maybe it's not such a bad result because in your own data only 6% of the customers accept the credit-card, so in your predictions (where you use less data) is very dificult that you find a customers that fits the profile to accept it. 

So if this was a real case maybe you should tell your boss to stop wasting resources on offering that card...

Anyway this project was about dealing with machine learning models and feature engineering and you clearly achieved that goal.

## Tableau
About tableau I think your visualizations are clear and quite self explanatory. The only thing to improve is that in question 7 in some of the boxes of the cross tables I can't see to wich cathegory belongs the box because they are too small, 


In summary, you've done really precise and meticulous job with this project, you can be reaaly proud of it .Well done.
